util/forms.py

```diff
@@ -0,0 +1,10 @@
+import json
+import os
+
+class Forms:
+    def get_form_body(file="1"):
+        try:
+            form_file = os.path.join(os.getcwd(), "forms", f"{file}.json")
+            return json.load(open(form_file, "r"))
+        except FileNotFoundError:
+            return {}
```

Comment:
Return an empty dict or allow the exception to be raised to the caller? IMHO allowing the exception up would be a better solution in the case where a form that's not expected is used